examples/example-webhooks-slack/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+This project shows how Asana webhooks can be used to send a celebratory message to Slack whenever a milestone is completed.
+It also contains examples of how to manage webhooks through the Python Asana client.
+
+## Webhooks server
+Start the server before creating any webhooks
+```
+ASANA_ACCESS_TOKEN=<your Asana PAT> SLACK_TOKEN=<your Slack API token> ./server.py 
+```
+
```

Comment:
It might be worth pointing out that the computer that this is running on needs to be DNS-addressable or have a public IP address that Asana can get to. This is often a point of confusion / friction for developers.